README.md

```diff
@@ -96,6 +97,7 @@ We love contributions! See our [CONTRIBUTING](docs/CONTRIBUTING.md) file for mor
       <td align="center" valign="top" width="14.28%"><a href="https://www.cthing.com"><img src="https://avatars.githubusercontent.com/u/989635?v=4?s=100" width="100px;" alt="Baron Roberts"/><br /><sub><b>Baron Roberts</b></sub></a><br /><a href="#code-baron1405" title="Code">💻</a> <a href="#ideas-baron1405" title="Ideas, Planning, & Feedback">🤔</a> <a href="#maintenance-baron1405" title="Maintenance">🚧</a> <a href="#review-baron1405" title="Reviewed Pull Requests">👀</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/vicky17d"><img src="https://avatars.githubusercontent.com/u/1669024?v=4?s=100" width="100px;" alt="vicky17d"/><br /><sub><b>vicky17d</b></sub></a><br /><a href="#code-vicky17d" title="Code">💻</a> <a href="#maintenance-vicky17d" title="Maintenance">🚧</a> <a href="#review-vicky17d" title="Reviewed Pull Requests">👀</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/opticyclic"><img src="https://avatars.githubusercontent.com/u/1222693?v=4?s=100" width="100px;" alt="opticyclic"/><br /><sub><b>opticyclic</b></sub></a><br /><a href="#content-opticyclic" title="Content">🖋</a> <a href="#example-opticyclic" title="Examples">💡</a> <a href="#ideas-opticyclic" title="Ideas, Planning, & Feedback">🤔</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/opticyclic"><img src="https://avatars.githubusercontent.com/u/16896724?v=4?s=100" width="100px;" alt="gehbiszumeis"/><br /><sub><b>gehbiszumeis</b></sub></a><br /><a href="#content-gehbizumeis" title="Content">🖋</a></td>
```

Comment:
```suggestion
```

Let's remove this for now - once this PR is merged in, I'll invoke the bot to open a PR to add you to the list 😄

Reply:
As promised 👉 https://github.com/ChrisCarini/jetbrains-ide-release-dates/pull/41